python/tvm/hybrid/_internal.py

```diff
@@ -0,0 +1,62 @@
+"""Internal utilities for parsing Python subset to HalideIR"""
+
```

Comment:
rename to util.py

when use it

from . import util as _util

then call

_util.prune_source